tests/fuzz/s2n_server_extensions_recv_test.c

```diff
@@ -56,17 +56,22 @@ int LLVMFuzzerInitialize(const uint8_t *buf, size_t len)
 
 int LLVMFuzzerTestOneInput(const uint8_t *buf, size_t len)
 {
-    for (int version = 0; version < s2n_array_len(TLS_VERSIONS); version++) {
+    /* We need at least one byte of input to set parameters */
+    if (len != 0) {
 
+        /* Setup */
         struct s2n_stuffer fuzz_stuffer = {0};
         GUARD(s2n_stuffer_alloc(&fuzz_stuffer, len + 1));
         GUARD(s2n_stuffer_write_bytes(&fuzz_stuffer, buf, len));
 
-        /* Setup */
         struct s2n_connection *client_conn = s2n_connection_new(S2N_CLIENT);
         notnull_check(client_conn);
-        client_conn->actual_protocol_version = TLS_VERSIONS[version];
-        client_conn->client_protocol_version = TLS_VERSIONS[version];
+
+        /* Pull a byte off the libfuzzer input and use it to set parameters */
+        uint8_t randval = 0;
+        GUARD(s2n_stuffer_read_uint8(&fuzz_stuffer, &randval));
+        client_conn->actual_protocol_version = TLS_VERSIONS[(randval & 0x0F) % s2n_array_len(TLS_VERSIONS)];
```

Comment:
Similar comment, we can fix this in a later PR, but we should have a `S2N_GET_BIT_SLICE(randval, 0, 4)` to get the bottom 4 bits, and a `S2N_GET_BIT_SLICE(randval, 4, 8)` to get the top 4 bits instead of doing the raw arithmetic inline.

Reply:
Sounds good, I'll tackle these in the future!